gecko/CMakeLists.txt

```
@@ -10,11 +10,28 @@
 # respectively.
 string(TOUPPER ${CONFIG_SOC_SERIES} SILABS_GECKO_DEVICE)
 
+# Remove any prefix from the board, e.g. efr32_radio_
+string(FIND ${CONFIG_BOARD} "brd" SILABS_GECKO_BRD_INDEX)
+
 set(SILABS_GECKO_PART_NUMBER ${CONFIG_SOC_PART_NUMBER})
 
 zephyr_include_directories(
   Device/SiliconLabs/${SILABS_GECKO_DEVICE}/Include
   emlib/inc
+  service/sleeptimer/src
+  service/sleeptimer/inc
+  service/sleeptimer/config
+  service/power_manager/src
+  service/power_manager/inc
+  service/power_manager/config
+  service/hfxo_manager/src
+  service/hfxo_manager/inc
+  service/hfxo_manager/config
+  common/inc
+  service/device_init/inc
+  service/device_init/config/s2
+  service/device_init/config/s2/sdid205
+  ${ZEPHYR_BASE}/boards/arm/${CONFIG_BOARD}
   )
```

I think `BOARD_ROOT` is a better fit here. But this variable is a list.
A potential solution could be:
```suggestion
  )
foreach(root ${BOARD_ROOT})
  if(IS_DIRECTORY "${root}/boards/arm/${CONFIG_BOARD}")
    zephyr_include_directories(${root}/boards/arm/${CONFIG_BOARD})
  endif()
endforeach()
```

---

Right, that will allow us to handle out-of-tree development. Thank you for this, I added the change to the PR.